sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -42,6 +43,11 @@ public enum ServiceVersion
         /// </summary>
         public ServiceVersion Version { get; }
 
+        /// <summary>
+        /// Gets the <see cref="CustomerProvidedKey"/> to be used when making requests.
+        /// </summary>
+        public CustomerProvidedKey? CustomerProvidedKey { get; }
```

Comment:
This property should be settable.  `FooClientOptions` is one of the few things we leave mutable (think changing Retry options, etc.).

Reply:
fixed.